sentences/de/homeassistant_HassGetCurrentDate.yaml

```
@@ -0,0 +1,7 @@
+language: de
+intents:
+  HassGetCurrentDate:
+    data:
+      - sentences:
+          - "<welche> (tag|datum) (ist|haben wir) heute"
+          - "(nenn[e]|gib|gebe) mir das[ heutige] datum"
```

```suggestion
          - "(nenn[e]|gib|gebe) mir das[ (heutige|aktuelle)] datum"
```
we should also allow "gebe mir das aktuelle datum"